Src/Eiffel/eiffel/interface/feature_i.e

```diff
@@ -2313,6 +2313,16 @@ feature -- Signature checking
 				then
 					error_handler.insert_error (create {VRVA}.make_invalid_type (Current, a_context_class, written_class, l_type))
 				end
+					-- A generic class can't have a once creation procedure
+				if is_once and then a_context_class.is_generic then
+					error_handler.insert_error (create {VGCC10}.make_invalid_context (Current, a_context_class, written_class))
+				end
+			else
+					-- For an inherited once feature, we check if it's part of the creations procedures.
+					-- In that case we raise an issue since once creation procedures must be immediates (not inherited)
+				if is_once and then	attached a_context_class.creators as l_creators and then l_creators.has (Current.feature_name) then
```

Comment:
Do we still have this restriction?

Reply:
Yes, we can't inherit once creation procedures.

Reply:
Hasn't that restriction been removed? (I guess, it was discussed at some meeting. I'm afraid, if it was not a technical meeting, we could not find a trace of the decision.)